React-Todo-list-main/src/App.js

```diff
@@ -2,35 +2,41 @@ import "./App.css";
 import { StatsigProvider } from "statsig-react";
 import { useState } from "react";
 import { TodoWrapper } from "./Components/TodoWrapper";
-
-const SS_CLIENT_SDK_KEY = "client-wWQtJlfJfEnLSIuP6QcnTATHyGVHUnJ2URTGLOVrwrl";
+import { SS_CLIENT_SDK_KEY } from "./Constant";
 
 /** Pass your StatsigUser into the StatsigProviderreturn (   
```

Comment:
Pass your StatsigUser into the StatsigProviderreturn -> Pass your StatsigUser into the StatsigProvider